func x(val: int) -> int {
    if val < 0 {
        throw "The value of a must be greater than or equal to 0."
    }
    return val
}

func main() -> nil {
    x(5)

    try {
        x(-2)
    } catch e {
        throw e
    }    
}
